### PR TITLE
feat(deploy): add config to deploy on bob network

### PR DESCRIPTION
**Pull request by juandahl**
### PR Description: Add Configuration for Bob Network Deployment

**Summary:**
This PR adds the necessary configuration to enable deployment on the Bob network.

**Changes:**
- **Deployment Configuration**: Added settings for deploying the application on the Bob network.

**Impact:**
- Expands deployment capabilities to include the Bob network.

### CODE REVIEW

File: helpers/types.ts [hunk 1]
```diff
@@ -59,6 +59,7 @@ export enum ePolygonNetwork {
 export enum eBobNetwork {
   main = "bob",
   testnet = "bob-sepolia",
+  testnet_new = "bob-sepolia-new",
```
Comment (matzapata) [inline helpers/types.ts]: Maybe I would do the opposit so `testnet_new` is `testnet` and `bob-sepolia` while we rename the old one to `testnet_old` or `testnet_deprecated` or so

File: tests/fork/pool.spec.ts [hunk 2]
```diff
@@ -41,11 +41,12 @@ makeSuite("Pool", (testEnv: TestEnv) => {
     );
   }
   
-  it("Supply Variable DAI", async () => {
+  it.only("Supply Variable DAI", async () => {
```
Comment (matzapata) [inline tests/fork/pool.spec.ts]: should this `only` be here?

**Review by matzapata — APPROVED** [review]
LGTM, one opinion about naming and a little question regarding a `only` in case it was forgotten

File: helpers/types.ts (hunk 1 above)
Comment (juandahl) [reply helpers/types.ts]: totally! Done

**Pull request merged**